deploy/examples/operator.yaml

```diff
@@ -44,6 +44,11 @@ data:
   # there is significant drop in read/write performance.
   # CSI_ENABLE_HOST_NETWORK: "true"
 
+  # Deprecation note: Rook uses "holder" pods to allow CSI to connect to the multus public network
+  # without needing hosts to the network. Holder pods are being deprecated. See issue for details:
+  # https://github.com/rook/rook/issues/13055. New Rook deployments should set this to "true".
+  CSI_DISABLE_HOLDER_PODS: "true"
```

Comment:
should we call it as `HOLDER_PODS` or `HOLDER_DAEMONSET` or just `HOLDER`?

Reply:
I thought about this and settled on "holder pods" (versus "daemonset" or just "holders") since most users seem to refer to the pods themselves and take on a pod-centric view.